configure.ac

```diff
@@ -2528,7 +2528,8 @@ if test x$host_win32 = xno; then
 	if test "x$with_tls" != "x__thread"; then
 		AC_MSG_RESULT(disabled)
 	elif test "x$cross_compiling" = "xyes"; then
-		AC_MSG_RESULT(cross compiling, assuming yes)
+		AC_MSG_RESULT(cross compiling, assuming no)
+		with_sigaltstack=no
```

Comment:
Could you add some comments around here? I don't understand how the condition relates to the body